### PR TITLE
For #4517: Updates visibility of fact items

### PR DESCRIPTION
**Pull request by sblatz**
---

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- [ ] **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- [ ] **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

File: components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/facts/ToolbarFacts.kt [hunk 1]
```diff
@@ -10,6 +10,13 @@ import mozilla.components.support.base.facts.Fact
 import mozilla.components.support.base.facts.collect
 import mozilla.components.ui.autocomplete.InlineAutocompleteEditText
 
+class ToolbarFacts {
+    object ToolbarItems {
+        const val TOOLBAR = "toolbar"
+        const val MENU = "menu"
+    }
+}
```
[the remark this answers is not in the document]
Comment (sblatz) [reply components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/facts/ToolbarFacts.kt]: Seems reasonable. I will rename :)

**Comment by sblatz** [comment]
@pocmo 
> We just landed a check in parallel that enforces public classes/methods to have kdoc documentation. Let's see if this blocks landing this PR:

Do you want me to add docs or do you want to use this as a guinea pig to see if it fails 😅

**Comment by pocmo** [comment]
> Do you want me to add docs or do you want to use this as a guinea pig to see if it fails 😅

I will never say no to docs. Feel free to update without any failure 😂

**Comment by sblatz** [comment]
Updated with docs, retrying merge.

bors retry

**Comment by sblatz** [comment]
@pocmo any idea why 

`getSnapshot() correctly clears the stores` fails for me? It's passing locally 🤔

**Comment by pocmo** [comment]
> getSnapshot() correctly clears the stores fails for me? It's passing locally 🤔

Do you still have a link to that failed build? Looks like after the last push it's not accessible from this PR anymore.

**Comment by pocmo** [comment]
Looks like it is gone.

bors r+

**Pull request merged**